test/functional/cmdLineTests/cmdLineTest_J9tests/j9tests_exclude.xml

```diff
@@ -49,6 +49,9 @@
 <exclude id="-Xthr:noSecondarySpinForObjectMonitors on command line" platform="aix.*" shouldFix="false"><reason>Only for platforms that three tier spin is enabled</reason></exclude>
 <exclude id="-Xthr:secondarySpinForObjectMonitors on command line" platform="aix.*" shouldFix="false"><reason>Only for platforms that three tier spin is enabled</reason></exclude>
 
+<exclude id="Verify -XX:+PortableSharedCache -XX:-PortableSharedCache behaves as expected with the portable shared cache feature disabled" platform=".*_aarch.*" shouldFix="true"><reason>unsupported on aarch</reason></exclude>
```

Comment:
Shouldn't this still work on aarch64 since the end results is the option disabled?

Reply:
Yes, the`-XX:-PortableSharedCache` test passes on aarch. Since PortableSharedCache is not supported on aarch, I disabled them both. If there is a value of running `-XX:-PortableSharedCache` on aarch, I can enable it.

Reply:
I don't expect it matters too much either way. Parsing the options should be common code across all platforms.